samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -0,0 +1,677 @@
+.. _broadcast_configuration_tool:
+
+Broadcast Configuration Tool
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Broadcast Configuration Tool sample implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>` and may act as an Auracast broadcaster if an Auracast compatible preset is used. It features a shell interface that allows the user to configure the broadcast source in many different ways.
+
+In this mode, transmitting broadcast audio happens using Broadcast Isochronous Stream (BIS) and Broadcast Isochronous Group (BIG).
+
+The following limitations apply to this sample:
+
+* Audio input: Pre-encoded LC3 data from an SD-card.
+* Maximum two BIG with four BIS streams each.
+
+.. _broadcast_configuration_tool_requirements:
+
+Requirements
+************
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Broadcast Configuration Tool sample is designed to be used only with the following hardware:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340_audio_dk_nrf5340
+
+.. note::
+   The sample supports PCA10121 revisions 1.0.0 or above.
+   The sample is also compatible with the following pre-launch revisions:
+
+   * Revisions 0.8.0 and above.
+
+.. _broadcast_configuration_tool_ui:
+
+Shell commands list
+*******************
+
+The Broadcast Configuration Tool uses a shell interface for all user interactions. The command used is
+
+  .. parsed-literal::
+     :class: highlight
+
+     bct
+
+When describing the commands ``<>`` means that the arguments is mandatory and ``[]`` means that the argument is optional.
+
+The following shell commands are available:
+
+start - Start broadcaster
+=========================
+
+The command starts the broadcaster with the current configuration. If no configuration is set, the broadcaster will not start. To view the current configuration, use the ``show`` command.
+The optional argument sets the index of the BIG to start.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct start [BIG_index]
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct start
+
+This will start any BIG that is configured.
```

Comment:
Please move all these lines (describing the examples) before the syntax of the actual example